### PR TITLE
Positive Test Result - swipe down moves the buttons

### PR DESCRIPTION
**Pull request by freshking**
## Description

Refactoring of the footer view mechanic to solve quite a few issues involving these buttons. Code is now a lot simpler and should be less error prone. 

For example there is no more need to set the height the footer manually. This is solved by using autolayout. 

## Link to Jira

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8302

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8388

### CODE REVIEW

**Review by pascal-brause — COMMENTED** [review]
Because you heavily change the behavior of the FooterView I cannot really say if this will breaking change something or not. 
Please test on small and big devices as much of the app as you can.

**Review by whiskey — APPROVED** [review]
Cool!

File: src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewController.swift [hunk 1]
```diff
@@ -233,17 +204,35 @@ class FooterViewController: UIViewController {
...
 			.store(in: &subscription)
 
 		viewModel.$backgroundColor
 			.receive(on: DispatchQueue.main.ocombine)
 			.assign(to: \.backgroundColor, on: view)
 			.store(in: &subscription)
 	}
+	
+	private func animateHeightChange() {
+		let animator = UIViewPropertyAnimator(duration: 0.35, curve: .easeInOut) { [weak self] in
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewController.swift:228]: I think UIView.animate would be enough here.

**Review by ArturFriesen — APPROVED** [review]
🚀

File: src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewController.swift (hunk 1 above)
Comment (freshking) [reply src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewController.swift:228]: I just used the same animation code that was used before this change.

**Review by pascal-brause — APPROVED** [review]

**Pull request merged**